docs/0.2-DRAFT/index.md

```diff
@@ -173,9 +172,25 @@ RO-Crate relies heavily on [schema.org](http://schema.org) using a constrained s
 Generally, the standard keys for [schema.org](http://schema.org) should be used. However, RO-Crate uses variant names for some elements, specifically:
 
 
-*   `File` is mapped to <http://schema.org/MediaObject> which was chosen as a compromise as it has many of the properties that are needed to describe a generic file. Future versions of schema.org or a research data extension may re-define `File`.
-*   `Journal` is mapped to <http://schema.org/Periodical>.
-*   `path` is mapped to <http://schema.org/contentUrl>. This property is used on some classes which do not strictly (yet) allow it in the official Schema.org.
+*   [File] is mapped to <http://schema.org/MediaObject> which was chosen as a compromise as it has many of the properties that are needed to describe a generic file. Future versions of schema.org or a research data extension may re-define `File`.
+*   [Journal] is mapped to <http://schema.org/Periodical>.
+*   [path] is mapped to <http://schema.org/contentUrl>. This property is used on some classes which do not strictly (yet) allow it in the official Schema.org.
+
+
+## Additional metadata standards
+
+`WorkflowSketch` - for an image depicting a workflow is from the ro-terms ontology <http://purl.org/ro/wf4ever#Sketch>, see the [documentation](http://wf4ever.github.io/ro/2016-01-28/roterms/#Sketch.)
+
+ RO-Crate also uses the _Portland Common Data Model_ ([PCDM](https://github.com/duraspace/pcdm/wiki)) and imports these terms:
+ 
+ [RepositoryObject](https://pcdm.org/2016/04/18/models#Object), 
+ [RepositoryCollection](https://pcdm.org/2016/04/18/models#Collection)
+ [RepositoryFile](https://pcdm.org/2016/04/18/models#Collection)
+ [hasMember](https://pcdm.org/2016/04/18/models#hasMember)
+ [hasFile](https://pcdm.org/2016/04/18/models#hasFile)
+
```

Comment:
as above, the aliases shouldn't be hyperlinked here and the format should be the same as above, i.e.
`RepositoryObject` is mapped to [https://pcdm.org/2016/04/18/models#Object](https://pcdm.org/2016/04/18/models#Object),